### PR TITLE
Factor duplicate unlocked wallet checks into new method 

### PR DESCRIPTION
**Pull request by ghubstan**
Response to [change request](https://github.com/bisq-network/bisq/pull/4299#discussion_r440769032) in PR [4299](https://github.com/bisq-network/bisq/pull/4299):

This PR should be reviewed/merged after PR [4309](https://github.com/bisq-network/bisq/pull/4309).

### CODE REVIEW

File: core/src/main/java/bisq/core/grpc/CoreWalletsService.java [hunk 1]
```diff
@@ -65,8 +65,7 @@ public long getAvailableBalance() {
         if (!walletsManager.areWalletsAvailable())
             throw new IllegalStateException("wallet is not yet available");
```
Comment (dmos62) [inline core/src/main/java/bisq/core/grpc/CoreWalletsService.java:66]: Following the same rationale, `verifyWalletsAreAvailable` should be created.

**Review by dmos62 — CHANGES_REQUESTED** [review]

File: core/src/main/java/bisq/core/grpc/CoreWalletsService.java (hunk 1 above)
Comment (ghubstan) [reply core/src/main/java/bisq/core/grpc/CoreWalletsService.java:66]: Makes sense, this change will be added to the latest PR in the current (and long) chain of PRs: [4329](https://github.com/bisq-network/bisq/pull/4329).

Comment (ghubstan) [reply core/src/main/java/bisq/core/grpc/CoreWalletsService.java:66]: The requested change is in commit e1fddfa, as part of PR [4329](https://github.com/bisq-network/bisq/pull/4329).

**Review by dmos62 — APPROVED** [review]

**Comment by dmos62** [comment]
utACK

**Review by sqrrm — APPROVED** [review]
utACK

**Pull request merged**